### PR TITLE
fixed bug 814, was changed and centered label on google button

### PR DESCRIPTION
**Pull request by skrebetsnataliia**
On "Fogot password page"  was centered and changed label  from "Sing in with google" to "Sing Up with google"
Wrong alignment  and label:
bug
Result:
9f693bf6-b994-4675-81b1-6b95a9b33ba6
https://github.com/ita-social-projects/GreenCity/issues/814

### CODE REVIEW

**Comment by yulkats** [comment]
please add a description

**Comment by skrebetsnataliia** [comment]
> Please add description with link and reviewers

Added description to this pull request

**Review by luchward — APPROVED** [review]

**Review by yulkats — APPROVED** [review]

**Review by Benerous — APPROVED** [review]

**Review by HryshkoAnton — APPROVED** [review]

File: src/app/component/auth/components/restore-password/restore-password.component.scss [hunk 1]
```diff
@@ -151,6 +151,7 @@ $inputHeight : 48px;
         line-height: 16px;
         text-align: center;
         letter-spacing: 0.21875px;
+        margin: 12px 92px 12px 62px;
```
Comment (mr-pavlenko) [inline src/app/component/auth/components/restore-password/restore-password.component.scss:154]: why these margins are different?

**Comment by skrebetsnataliia** [comment]
Bug will be fixed in this pull rquest https://github.com/ita-social-projects/GreenCityClient/pull/632.

**Pull request closed without merging**